lang/cs/Org.Apache.REEF.IO.Tests/TestFileDataSet.cs

```diff
@@ -0,0 +1,275 @@
+                .Build();
+
+            var dataSet = TangFactory.GetTang()
+                .NewInjector(c)
+                .GetInstance<IPartitionedDataSet>();
+            
+            int count = 0;
+            foreach (var partitionDescriptor in dataSet)
+            {
+                var partition =
+                    TangFactory.GetTang()
+                        .NewInjector(partitionDescriptor.GetPartitionConfiguration())
+                        .GetInstance<IPartition<IEnumerable<byte>>>();
+
+                var e = partition.GetPartitionHandle();
+                foreach (var v in e)
+                {
+                    Console.WriteLine(v);
+                    count++;
+                }
+            }
+            Assert.AreEqual(count, 7);
+        }
+
+        /// <remarks>
+        /// This test is to use a RowSerializer. Row is a class at client side. 
+        /// </remarks>
+        [TestMethod]
+        public void TestWithRowDeserializer()
+        {
+            const string tempFileName1 = "REEF.TestLocalFileSystem1.tmp";
+            const string tempFileName2 = "REEF.TestLocalFileSystem2.tmp";
+            var sourceFilePath1 = Path.Combine(Path.GetTempPath(), tempFileName1);
+            var sourceFilePath2 = Path.Combine(Path.GetTempPath(), tempFileName2);
+
+            MakeLocalTestFile(sourceFilePath1, new byte[] {111, 112, 113});
+            MakeLocalTestFile(sourceFilePath2, new byte[] { 114, 115 });
+
+            var dataSet = TangFactory.GetTang()
+                .NewInjector(FileSystemPartitionConfiguration<Row>.ConfigurationModule
+                    .Set(FileSystemPartitionConfiguration<Row>.FilePaths, sourceFilePath1)
+                    .Set(FileSystemPartitionConfiguration<Row>.FilePaths, sourceFilePath2)
+                    .Set(FileSystemPartitionConfiguration<Row>.FileSerializerConfig, GetRowSerilizerConfigString())
+                    .Build())
+                .GetInstance<IPartitionedDataSet>();
+
+            int count = 0;
+            foreach (var partitionDescriptor in dataSet)
+            {
+                var partition =
+                    TangFactory.GetTang()
+                        .NewInjector(partitionDescriptor.GetPartitionConfiguration())
+                        .GetInstance<IPartition<IEnumerable<Row>>>();
+
+                IEnumerable<Row> e = partition.GetPartitionHandle();
+
+                foreach (var row in e)
+                {
+                    Console.WriteLine(row.GetValue());
+                    count++;
+                }
+            }
+            Assert.AreEqual(count, 5);
+        }
+
+        private void MakeLocalTestFile(string filePath, byte[] bytes)
+        {
+            if (File.Exists(filePath))
+            {
+                File.Delete(filePath);
+            }
+
+            using (var s = File.Create(filePath))
+            {
+                foreach (var b in bytes)
+                    s.WriteByte(b);
```

Comment:
yes, I just want to simulate something that writes one by one and later read one by one.